lldb/test/API/functionalities/data-formatter/data-formatter-stl/libcxx/multimap/TestDataFormatterLibccMultiMap.py

```diff
@@ -19,6 +19,7 @@ def setUp(self):
         TestBase.setUp(self)
         self.namespace = 'std'
 
+    @skipIf(bugnumber="rdar://74106716")
```

Comment:
You can XFAIL instead of skip this one if you limit it to the gmodules debuginfo.

```
 @expectedFailureAll(debug_info="gmodules")
```

(The other tests here are actually quite important :) )

Reply:
This test passes for me locally, so should it be skipped under gmodules?

Reply:
Hm, good question. The problem is that the Swift build system is replacing the libc++ we're testing against with the System one (instead of the libc++ of the current llvm-project commit), so depending on what system libc++ you have this test might fail or pass. I actually thought that it passes with the upstream libc++, but if there are also downstream libc++ that makes this test pass then we can't really XFAIL it. So let's just skip the modules variation instead.

Reply:
Cool, I believe the current revision does that.